homeassistant/components/opower/__init__.py

```diff
@@ -2,21 +2,31 @@
 
 from __future__ import annotations
 
+import logging
+
 from homeassistant.config_entries import ConfigEntry
 from homeassistant.const import Platform
 from homeassistant.core import HomeAssistant
+from homeassistant.exceptions import ConfigEntryNotReady
 
 from .const import DOMAIN
 from .coordinator import OpowerCoordinator
 
 PLATFORMS: list[Platform] = [Platform.SENSOR]
 
+_LOGGER = logging.getLogger(__name__)
+
 
 async def async_setup_entry(hass: HomeAssistant, entry: ConfigEntry) -> bool:
     """Set up Opower from a config entry."""
 
     coordinator = OpowerCoordinator(hass, entry.data)
-    await coordinator.async_config_entry_first_refresh()
+    try:
+        await coordinator.async_config_entry_first_refresh()
+    except ConfigEntryNotReady:
+        _LOGGER.exception("Unable to setup Opower")
```

Comment:
Why print the whole stack trace. Shouldn't the message of ConfigEntryNotReady be enough to identify the cause?